pandas/core/indexes/base.py

```diff
@@ -1404,6 +1416,16 @@ def summary(self, name=None):
             name = type(self).__name__
         return '%s: %s entries%s' % (name, len(self), index_summary)
 
+    def summary(self, name=None):
+        """
+        Return a summarized representation
+        .. deprecated:: 0.23.0
+           No longer supported
```

Comment:
you don't need the 'No longer supported'

Reply:
@jreback - This has been removed per your request. Update has been pushed.